docs/submit/osg-flock.md

```diff
@@ -83,26 +73,21 @@ You will need information like the hostname, and the administrative and security
 Follow the [general registration instructions](../common/registration.md#new-resources).
 For historical reasons, the service type is `Submit Node`.
 
-
-### Choose your authentication method
-There are two options for authentication of your access point to the OSG: GSI and pool password.
-Of these, GSI is the recommended method, but it will require obtaining a host certificate for your access point.
-See our [documentation](../security/host-certs.md) for instructions on how to request and install host certificates.
-
-If you are unable to obtain a host certificate, use the "pool password" authentication method,
-[described in the Configuring Authentication section](#configuring-authentication).
-
-
 ### Request to be allowed to flock to OSG
-OSG staff will need to add your access point to the list of hosts that flocked jobs are accepted from.
+OSG staff will need to add your access point to the list of hosts that flocked jobs are accepted from,
+and provide a HTCondor IDTOKEN so that your new access point can authenticate with OSG.
 Send email to <help@opensciencegrid.org> with the hostname of your access point and request to be added to the list.
 
 
 Installing Required Software
 ----------------------------
 Flocking requires HTCondor software as well as software for reporting to the OSG accounting system.
-OSG provides a convenience RPM that installs all required packages.
-To install, run:
+Start by setting up the OSG YUM repositories following the
+[Installing Yum Repositories](../common/yum.md). __Note that you have to use OSG 3.6__. Earlier
+versions will not work.
```

Comment:
Why is this the case? We ship 8.9.11 in 3.5 upcoming. I'm a little hesitant to push 3.6 right now but as long as we know that this works, I'm ok with telling folks to use 3.6

Reply:
3.6 is better long term here. That is what I did the testing with.

Reply:
Ok sure. Could you make this note an [admonition](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#supported-types) to highlight it further?

Reply:
Yes, but let's get this in place and I can take a second pass on it later